### PR TITLE
Pass calendar ID for creating the event

### PR DESCRIPTION
**Pull request by harshith-venkatesh**
### What is the change?

Provide a small description of what did you change and provide the reference to the issue ticket.

### Is it bug?

- Steps to repro
- Expected
- Actual

### \*Dev Tested?

- [ ] Yes
- [ ] No

#### Platforms

- [ ] Web

#### Browsers

- [ ] Chrome
- [ ] Safari
- [ ] Firefox

### Before / After Change Screenshots

> For visual or interaction changes. Can be video / screenshot.

### CODE REVIEW

File: app/routes/index.tsx [hunk 1]
```diff
@@ -30,7 +31,7 @@ export const loader: LoaderFunction = async ({ request }) => {
       ENV: baseUrls,
       error: 'Unable to fetch the user details, please login',
     }));
-
+    console.log({data})
```
Comment (iRohitGaur) [inline app/routes/index.tsx:34]: please remove

File: app/routes/index.tsx [hunk 2]
```diff
@@ -74,15 +74,15 @@ export const loader: LoaderFunction = async ({ request }) => {
...
+  const { events, error, calendarId } = useLoaderData();
   useEffect(() => {
     if (error === null && events.length > 0) {
       // TODO: show a  different message if events are not present in the given date range
-
+      sessionStorage.setItem('calendarId', calendarId)
       setEvents([parseEvents(events)][0]);
     } else if (error === null && events.length === 0) {
       // TODO: discuss regarding display of user message in case of no events
+      sessionStorage.setItem('calendarId', calendarId)
```
Comment (iRohitGaur) [inline app/routes/index.tsx:85]: we can manage this using zustand instead of using session storage

**Review by iRohitGaur — APPROVED** [review]

File: app/routes/index.tsx [hunk 3]
```diff
@@ -74,15 +74,15 @@ export const loader: LoaderFunction = async ({ request }) => {
...
 function CalendarPage() {
   const { setEvents, events: eventList, view } = useStore((state) => state);
-  const { events, error } = useLoaderData();
-
+  const { events, error, calendarId } = useLoaderData();
   useEffect(() => {
     if (error === null && events.length > 0) {
       // TODO: show a  different message if events are not present in the given date range
-
+      sessionStorage.setItem('calendarId', calendarId)
```
Comment (isVivek99) [inline app/routes/index.tsx:81]: any reason to use calendarId instead of the global storage?

Comment (harshith-venkatesh) [reply app/routes/index.tsx]: Have used zustand to store the calendar ID now

**Review by iRohitGaur — APPROVED** [review]

**Pull request merged**